src/osa/scripts/gainsel_webmaker.py

```diff
@@ -0,0 +1,160 @@
+from pathlib import Path
+from textwrap import dedent
+from astropy.table import Table
+from datetime import datetime, timedelta
+from osa.utils.utils import date_to_dir, date_to_iso
+from osa.configs.config import cfg
+from osa.paths import DEFAULT_CFG
+from osa.nightsummary.nightsummary import run_summary_table
+import numpy as np
+import pandas as pd
+from argparse import ArgumentParser
+from osa.configs import options
+
+def valid_date(string):
+    """Check if the string is a valid date and return a datetime object."""
+    return datetime.strptime(string, "%Y%m%d")
+
+
+common_parser = ArgumentParser(add_help=False)
+common_parser.add_argument(
+    "-c",
+    "--config",
+    type=Path,
+    default=DEFAULT_CFG,
+    help="Use specific config file [default configs/sequencer.cfg]",
+)
+common_parser.add_argument(
+    "-d",
+    "--date",
+    help="Date (YYYYMMDD) of the start of the night",
+    type=valid_date,
+)
+
+def html_content(body: str, date: str) -> str:
```

Comment:
re use from defined function somewhere else

Reply:
The function cannot be reused because the HTML is not the same as sequencer webmaker.

Reply:
cannot be adapted from the sequencer webmaker (changing the title of the HTML through function argument or something like that) ?